lib/mparticle_flutter_sdk_web.dart

```diff
@@ -260,6 +261,119 @@ class MparticleFlutterSdkWeb {
           print(
               'You included only a startTime or an endTime, but not both. Please include BOTH a startTime and an endTime, or let mParticle calculate both for you');
         }
+        break;
+      case 'logCommerceEvent':
+        var commerceEvent = call.arguments['commerceEvent'];
+
+        var customAttributes = commerceEvent['customAttributes'];
+        if (customAttributes == null) {
+          customAttributes = {};
+        }
+
+        var customFlags = commerceEvent['customFlags'];
+        if (customFlags == null) {
+          customFlags = {};
+        }
+
+        String? currency = commerceEvent['currency'];
+        if (currency != null) {
+          mpCommerce.callMethod('setCurrencyCode', [currency]);
+        }
+
+        var transactionAttributes = {};
+        if (commerceEvent['transactionAttributes'] != null) {
+          transactionAttributes = commerceEvent['transactionAttributes'];
```

Comment:
this is a super nit-pick, but it's eating away at me that you flipped the logic here from customAttributes and customFlags above 🤣